nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/api/ProcessGroupResource.java

```diff
@@ -1969,8 +1970,12 @@ public Response createProcessGroup(
             @ApiParam(
                     value = "The process group configuration details.",
                     required = true
-        ) final ProcessGroupEntity requestProcessGroupEntity) {
-
+            )
+            final ProcessGroupEntity requestProcessGroupEntity,
+            @QueryParam("keepExistingParameterContext")
+            @DefaultValue("true")
+            final boolean keepExistingParameterContext
```

Comment:
In terms of supporting potential API evolution, what do you think of making this an `enum` named `ParameterContextHandlingStrategy`? The default value could be `KEEP_EXISTING`, and the alternative could be `REPLACE` or similar? That would provide more flexibility and also help clarifying the options.